ipv4_address.go

```diff
@@ -21,47 +21,24 @@ package hedera
  */
 
 type IPv4Address struct {
```

Comment:
I think this stuct can be removed as well, it just holds the `address []byte` and does not really do anything.
For example the `endpoint` should call `endpoint.address.address` which is weird.

Reply:
True, lets remove.